variant_remapping_tools/tests/test_reads_to_remapped_variants.py

```diff
@@ -1,10 +1,11 @@
 import os
+from collections import Counter
 from unittest.mock import Mock, patch
 
 import pysam
 from unittest import TestCase
 from variant_remapping_tools.reads_to_remapped_variants import fetch_bases, process_bam_file, \
-    calculate_new_variant_definition, _order_reads, link_supplementary
+    calculate_new_variant_definition, _order_reads, link_supplementary, pass_aligned_filtering, group_reads
```

Comment:
It was already like this before the change, but I'll point it out anyway: if `_order_reads` is being imported from that module, it shouldn't be a private method (starting with `_`)

Reply:
Yes that didn't make sense I changed it